scripts/run-pg-docker.sh

```diff
@@ -7,6 +7,9 @@ echo "Creating network $network_name..."
 docker network create $network_name
 echo "Created network $network_name"
 
+echo "Creating /tmp/rants-data for database volume"
+mkdirp /tmp/rants-data
```

Comment:
this should probably be `mkdir -p /tmp/rants-data`. I don't have `mkdirp` installed globally :smile:

Reply:
Heh. Yes. That's what I meant.